benchmark/benchmark.hpp

```diff
@@ -141,19 +219,28 @@ template <typename Pool> class pool_allocator : public allocator_interface {
 
 template <typename Size, typename Allocator>
 struct benchmark_interface : public benchmark::Fixture {
-    void SetUp(::benchmark::State &state) {
-        int argPos = alloc_size.SetUp(state, 0);
-        allocator.SetUp(state, argPos);
+    int parseArgs(::benchmark::State &state, int argPos) {
+        Size generator;
+        argPos = generator.SetUp(state, argPos);
+        argPos = allocator.SetUp(state, argPos);
+        alloc_sizes.resize(state.threads());
+        for (auto &i : alloc_sizes) {
+            i = generator;
+        }
+        return argPos;
```

Comment:
This return value is not used

Reply:
It is used in class that inherits this one.